source/guides/guides/retry-ability.md

```diff
@@ -2,41 +2,55 @@
 title: Retry-ability
 ---
 
-Retry-ability is the core part of Cypress Test Runner that allows it to test dynamic web applications. Like a good car transmission, it usually works without you noticing it, but understanding how it works will help you write faster tests with fewer run-time surprises.
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+- How Cypress retries commands and assertions
+- When commands are retried and when they are not
+- How to address some situations of flaky tests
+{% endnote %}
+
+A core feature of Cypress that assists with testing dynamic web applications is retry-ability. Like a good transmission in a car, it usually works without you noticing it. But understanding how it works will help you write faster tests with fewer run-time surprises.
```

Comment:
thank you for leaving in my car transmission analogy!

Reply:
It is good! 😆